Kernel/Devices/Audio/AC97/AC97.cpp

```diff
@@ -35,6 +35,21 @@ UNMAP_AFTER_INIT ErrorOr<NonnullRefPtr<AudioController>> AC97::create(PCI::Devic
 UNMAP_AFTER_INIT ErrorOr<bool> AC97::probe(PCI::DeviceIdentifier const& device_identifier)
```

Comment:
Commit title should be "Kernel/Audio: Fail AC97 probe if no good BAR1 is found".

Reply:
Done